Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -502,4 +526,33 @@ class NewsTableViewController: UITableViewController,
         let activity = ShareCanillitapActivity(withNews: n)
         return [activity]
     }
+    
+    // MARK: UIViewControllerPreviewingDelegate
+    func previewingContext(_ previewingContext: UIViewControllerPreviewing,
+                           viewControllerForLocation location: CGPoint) -> UIViewController? {
+        
+        guard let indexPath = tableView.indexPathForRow(at: tableView.convert(location, from: view)) else {
+            return nil
+        }
+        
+        let news = filteredNewsViewModels[indexPath.row].news
+        
+        let storyboard = UIStoryboard(name: "NewsPreview", bundle: nil)
+        let vc: NewsPreviewViewController? = storyboard.instantiateInitialViewController() as? NewsPreviewViewController
+        vc?.news = news
+        vc?.newsViewController = self
+        vc?.preferredContentSize = CGSize(width: 300, height: 300)
+        selectedNews = news
+        return vc
+    }
+    
+    func previewingContext(_ previewingContext: UIViewControllerPreviewing, commit viewControllerToCommit: UIViewController) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/ba933f4049a44fb3a8fc655051f4e975dcf6eb8e/Headlines/src/Controllers/NewsTableViewController.swift#L549"></span>Line should be 120 characters or less: currently 126 characters</td>
    </tr>
  </tbody>
</table>